### PR TITLE
Created a new pack name - "Integrations & Incidents Health Check"

### PR DESCRIPTION
**Pull request by altmannyarden**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/14886

## Description
A new pack to run tests and gets details for failed enabled integrations and open incidents.

## Screenshots
image

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by yaron-libman** [comment]
reviewed.

File: Packs/IntegrationsAndIncidentsHealthCheck/Scripts/IntegrationsCheck-Widget-IntegrationsCategory/IntegrationsCheck-Widget-IntegrationsCategory.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+runas: DBotWeakRole
+runonce: false
+script: ''
+scripttarget: 0
+subtype: python3
+tags:
+- widget
+- failedInstances
+type: python
+fromversion: 5.0.0
```
Comment (teizenman) [inline Packs/IntegrationsAndIncidentsHealthCheck/Scripts/IntegrationsCheck-Widget-IntegrationsCategory/IntegrationsCheck-Widget-IntegrationsCategory.yml:18]: same here

File: Packs/IntegrationsAndIncidentsHealthCheck/Scripts/IntegrationsCheck-Widget-IntegrationsErrorsInfo/IntegrationsCheck-Widget-IntegrationsErrorsInfo.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+runas: DBotWeakRole
+runonce: false
+script: ''
+scripttarget: 0
+subtype: python3
+tags:
+- widget
+- failedInstances
+type: python
+fromversion: 5.0.0
```
Comment (teizenman) [inline Packs/IntegrationsAndIncidentsHealthCheck/Scripts/IntegrationsCheck-Widget-IntegrationsErrorsInfo/IntegrationsCheck-Widget-IntegrationsErrorsInfo.yml:18]: same here

File: Packs/IntegrationsAndIncidentsHealthCheck/Scripts/IntegrationsCheck-Widget-NumberChecked/IntegrationsCheck-Widget-NumberChecked.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+runas: DBotWeakRole
+runonce: false
+script: ''
+scripttarget: 0
+subtype: python3
+tags:
+- widget
+- failedInstances
+type: python
+fromversion: 5.0.0
```
Comment (teizenman) [inline Packs/IntegrationsAndIncidentsHealthCheck/Scripts/IntegrationsCheck-Widget-NumberChecked/IntegrationsCheck-Widget-NumberChecked.yml:18]: same here

File: Packs/IntegrationsAndIncidentsHealthCheck/Scripts/IntegrationsCheck-Widget-NumberFailingInstances/IntegrationsCheck-Widget-NumberFailingInstances.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+runas: DBotWeakRole
+runonce: false
+script: ''
+scripttarget: 0
+subtype: python3
+tags:
+- widget
+- failedInstances
+type: python
+fromversion: 5.0.0
```
Comment (teizenman) [inline Packs/IntegrationsAndIncidentsHealthCheck/Scripts/IntegrationsCheck-Widget-NumberFailingInstances/IntegrationsCheck-Widget-NumberFailingInstances.yml:18]: same here

**Review by teizenman — COMMENTED** [review]
Committing most of the changed myself

**Review by michalgold — APPROVED** [review]

**Pull request merged**